src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -260,26 +260,49 @@ internal static object GetDependencyPropertyValueInternal(DependencyObject owner
 			return dp == null ? null : owner.GetValue(dp);
 		}
 
-		internal static string SetDependencyPropertyValueInternal(DependencyObject owner, string dependencyPropertyName, string value)
+		/// <summary>
+		/// Sets the specified dependency property value using the format "name|value"
+		/// </summary>
+		/// <param name="dependencyPropertyNameAndValue">The name and value of the property</param>
+		/// <returns>The currenty set value at the Local precedence</returns>
+		/// <remarks>
+		/// The signature of this method was chosen to work around a limitation of Xamarin.UITest with regards to
+		/// parameters passing on iOS, where the number of parameters follows a unconventional set of rules. Using
+		/// a single parameter with a simple delimitation format fits all platforms with little overhead.
+		/// </remarks>
+		internal static string SetDependencyPropertyValueInternal(DependencyObject owner, string dependencyPropertyNameAndValue)
```

Comment:
Would be good to have a usage example in the xml doc

Reply:
It's in the summary, ` using the format "name|value"`